src/Compilers/Core/CodeAnalysisTest/Text/TextChangeTests.cs

```diff
@@ -880,6 +902,221 @@ public void TestMergeChanges_NoMiddleMan()
+            Assert.Equal("4bbaa5", originalText.WithChanges(changes).ToString());
+        }
+
+        [Fact]
+        public void Fuzz_7()
+        {
+            var originalText = SourceText.From("01234567");
+            var change1 = originalText.WithChanges(new TextChange(new TextSpan(0, 1), "aaaaa"), new TextChange(new TextSpan(3, 1), "aaaa"), new TextChange(new TextSpan(6, 1), "aaaaa"));
+            var change2 = change1.WithChanges(new TextChange(new TextSpan(0, 0), "b"), new TextChange(new TextSpan(2, 0), "b"), new TextChange(new TextSpan(3, 4), "bbbbb"), new TextChange(new TextSpan(9, 5), "bbbbb"), new TextChange(new TextSpan(15, 3), ""));
+            Assert.Equal("aaaaa12aaaa45aaaaa7", change1.ToString());
+            Assert.Equal("baababbbbbaabbbbba7", change2.ToString());
+
+
+            var changes = change2.GetTextChanges(originalText);
+            Assert.Equal("baababbbbbaabbbbba7", originalText.WithChanges(changes).ToString());
+        }
+
+        [Fact]
+        [WorkItem(47234, "https://github.com/dotnet/roslyn/issues/47234")]
+        public void Fuzz_10()
+        {
+            var originalText = SourceText.From("01234");
+            var change1 = originalText.WithChanges(new TextChange(new TextSpan(0, 1), "a"));
+            var change2 = change1.WithChanges(new TextChange(new TextSpan(0, 1), "b"), new TextChange(new TextSpan(2, 2), "b"));
+            Assert.Equal("a1234", change1.ToString());
+            Assert.Equal("b1b4", change2.ToString());
+
+            var changes = change2.GetTextChanges(originalText);
+            Assert.Equal("b1b4", originalText.WithChanges(changes).ToString());
+        }
+
+        [Fact]
+        [WorkItem(47234, "https://github.com/dotnet/roslyn/issues/47234")]
+        public void Fuzz_23()
+        {
+            var originalText = SourceText.From("01234");
+            var change1 = originalText.WithChanges(new TextChange(new TextSpan(0, 1), "aa"));
+            var change2 = change1.WithChanges(new TextChange(new TextSpan(0, 0), "b"), new TextChange(new TextSpan(1, 2), "b"));
+            Assert.Equal("aa1234", change1.ToString());
+            Assert.Equal("bab234", change2.ToString());
+
+            var changes = change2.GetTextChanges(originalText);
+            Assert.Equal("bab234", originalText.WithChanges(changes).ToString());
+        }
+
+        [Fact]
+        public void Fuzz_32()
+        {
+            var originalText = SourceText.From("012345");
+            var change1 = originalText.WithChanges(new TextChange(new TextSpan(0, 2), "a"), new TextChange(new TextSpan(3, 2), "a"));
+            var change2 = change1.WithChanges(new TextChange(new TextSpan(0, 3), "bbb"));
+            Assert.Equal("a2a5", change1.ToString());
+            Assert.Equal("bbb5", change2.ToString());
+
+            var changes = change2.GetTextChanges(originalText);
+            Assert.Equal("bbb5", originalText.WithChanges(changes).ToString());
+        }
```

Comment:
where did 24 through 31 go?

Reply:
Addressed in https://github.com/dotnet/roslyn/pull/48420#discussion_r502549529